tests/e2e/common.go

```diff
@@ -79,13 +79,19 @@ func registerFlags(td *OsmTestData) {
 	flag.BoolVar(&td.kindCluster, "kindCluster", false, "Creates kind cluster")
 	flag.StringVar(&td.clusterName, "kindClusterName", "osm-e2e", "Name of the Kind cluster to be created")
 	flag.BoolVar(&td.cleanupKindCluster, "cleanupKindCluster", true, "Cleanup kind cluster upon exit")
-	flag.BoolVar(&td.cleanupKindClusterBetweenTests, "cleanupKindClusterBetweenTests", true, "Cleanup kind cluster between tests")
+	flag.BoolVar(&td.cleanupKindClusterBetweenTests, "cleanupKindClusterBetweenTests", false, "Cleanup kind cluster between tests")
 
 	flag.StringVar(&td.ctrRegistryServer, "ctrRegistry", os.Getenv("CTR_REGISTRY"), "Container registry")
 	flag.StringVar(&td.ctrRegistryUser, "ctrRegistryUser", os.Getenv("CTR_REGISTRY_USER"), "Container registry")
 	flag.StringVar(&td.ctrRegistryPassword, "ctrRegistrySecret", os.Getenv("CTR_REGISTRY_PASSWORD"), "Container registry secret")
 
-	flag.StringVar(&td.osmImageTag, "osmImageTag", "latest", "OSM image tag")
+	flag.StringVar(&td.osmImageTag, "osmImageTag", func() string {
+		tmp := os.Getenv("CTR_TAG")
+		if len(tmp) != 0 {
+			return tmp
+		}
+		return "latest"
+	}(), "OSM image tag")
```

Comment:
We could move this https://github.com/openservicemesh/osm/blob/c98e2556ca03ec7a0774e38066ece4a16f53ebb2/demo/cmd/common/books.go#L96-L103 into some util package and use it.

Reply:
makes sense
will do